grpc/src/client/load_balancing/child_manager.rs

```diff
@@ -305,3 +460,32 @@ impl WorkScheduler for ChildWorkScheduler {
         }
     }
 }
+
+struct RoundRobinPicker {
+    pickers: Vec<Arc<dyn Picker>>,
+    next: AtomicUsize,
+}
+
+impl RoundRobinPicker {
+    fn new() -> Self {
+        Self {
+            pickers: vec![],
+            next: AtomicUsize::new(0),
+        }
+    }
+
+    fn add_picker(&mut self, picker: Arc<dyn Picker>) {
+        self.pickers.push(picker);
+    }
```

Comment:
Can we make it so all the pickers are known when `new` is called instead?

Ideally we set `next` initially to a random value in the range of the number of entries.